readme.md

```diff
@@ -127,6 +127,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [AutoIt](https://github.com/J2TeaM/awesome-AutoIt)
 - [Crystal](https://github.com/veelenga/awesome-crystal)
 - [TypeScript](https://github.com/dzharii/awesome-typescript)
+- [Frege](https://github.com/sfischer13/awesome-frege)
```

Comment:
Add a succinct description. See other entries with description on how to format it.